src/main/java/core/basesyntax/Display.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Display {
```

Comment:
'Display' is not a good name for interface. Better name it Drawable or Displayable
```suggestion
public interface Drawable {
```